docs/ko/docs/tutorial/response-status-code.md

```diff
@@ -0,0 +1,89 @@
+# 응답 상태 코드
+
+응답 모델(response model)과 같은 방법으로, 어떤 *경로 동작*이든 `status_code` 매개변수를 사용하여 응답에 대한 HTTP 상태 코드를 선언할 수 있습니다.
+
+* `@app.get()`
+* `@app.post()`
+* `@app.put()`
+* `@app.delete()`
+* 기타
+
+```Python hl_lines="6" 
+{!../../../docs_src/response_status_code/tutorial001.py!}
+```
+
+!!! note "참고" 
+    `status_code` 는 "데코레이터" 메소드(`get`, `post` 등)의 매개변수입니다. 모든 매개변수들과 본문(body)처럼 *경로 동작 함수*가 아닙니다.
+
+`status_code` 매개변수는 HTTP 상태 코드를 숫자로 입력받습니다.
+
+!!! info "정보"
+    `status_code` 는 Python의 `http.HTTPStatus` 와 같은 `IntEnum` 을 입력받을 수도 있습니다.
+
+`status_code` 매개변수는:
+
+* 응답에서 해당 상태 코드를 반환합니다.
+* 상태 코드를 OpenAPI 스키마(및 사용자 인터페이스)에 문서화 합니다.
+
+<img src="https://fastapi.tiangolo.com/img/tutorial/response-status-code/image01.png">
+
+!!! note "참고" 
+    어떤 응답 코드들은 해당 응답에 본문(body)이 없다는 것을 의미하기도 합니다(다음 항목 참고).
+
+    이에 따라 FastAPI는 응답 본문이 없음을 명시하는 OpenAPI를 생성합니다.
+
+## HTTP 상태 코드에 대하여
+
+!!! note "참고"
+    만약 HTTP 상태 코드에 대하여 이미 알고있다면, 다음 항목으로 넘어가십시오.
+
+HTTP는 세자리의 숫자 상태 코드를 응답의 일부로 전송합니다.
+
+이 상태 코드들은 각자를 식별할 수 있도록 지정된 이름이 있으나, 중요한 것은 숫자 코드입니다.
+
+요약하자면:
+
+* `**1xx**` 상태 코드는 "정보"용입니다. 이들은 직접적으로는 잘 사용되지는 않습니다. 이 상태 코드를 갖는 응답들은 본문을 가질 수 없습니다.
+* `**2xx**` 상태 코드는 "성공적인" 응답을 위해 사용됩니다. 가장 많이 사용되는 유형입니다.
+    * `200` 은 디폴트 상태 코드로, 모든 것이 "성공적임(OK)"을 의미합니다.
+    * 다른 예로는 `201` "생성됨(Created)"이 있습니다. 일반적으로 데이터베이스에 새로운 레코드를 생성한 후 사용합니다.
+    * 단, `204` "내용 없음(No Content)"은 특별한 경우입니다. 이것은 클라이언트에게 반환할 내용이 없는 경우 사용합니다. 따라서 응답은 본문을 가질 수 없습니다.
+* `**3xx**` 상태 코드는 "리다이렉션(Redirection)"용입니다. 본문을 가질 수 없는 `304` "수정되지 않음(Not Modified)"을 제외하고, 이 상태 코드를 갖는 응답에는 본문이 있을 수도, 없을 수도 있습니다.
+* `**4xx**` 상태 코드는 "클라이언트 오류(Client error)" 응답을 위해 사용됩니다. 이것은 아마 가장 많이 사용하게 될 두번째 유형입니다.
+    * 일례로 `404` 는 "찾을 수 없음(Not Found)" 응답을 위해 사용합니다.
+    * 일반적인 클라이언트 오류의 경우 `400` 을 사용할 수 있습니다.
+* `**5xx**` 상태 코드는 서버 오류에 사용됩니다. 이것들을 직접 사용할 일은 거의 없습니다. 어플리케이션 코드나 서버의 일부에서 문제가 발생하면 자동으로 이들 상태 코드 중 하나를 반환합니다.
```

Comment:
```suggestion
* `**5xx**` 상태 코드는 서버 오류에 사용됩니다. 이것들을 직접 사용할 일은 거의 없습니다. 응용 프로그램 코드나 서버의 일부에서 문제가 발생하면 자동으로 이들 상태 코드 중 하나를 반환합니다.
```
If you check [파이썬 한국어 번역](https://www.flowdas.com/pages/python-docs-ko.html) and search word, `application`, it is translated as `응용 프로그램` and because we follows the rules of that guide (check of it on #2017 ).